core/pom.xml

```diff
@@ -532,7 +533,12 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-crypto</artifactId>
     </dependency>
-
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
```

Comment:
Related context: https://github.com/apache/spark/pull/37972/files#r989470894

Reply:
Can we have individual PR for this?

Reply:
This is for the core module. Do you mean put the changes of the other pom files into an individual PR?

Reply:
Created https://github.com/apache/spark/pull/38783